.travis.yml

```diff
@@ -17,6 +21,10 @@ matrix:
     - jdk: oraclejdk8
       scala: 2.12.9
       env: COMMAND=ci-jvm-all
+    # Scala 2.12, JVM, JDK 11
+    - jdk: openjdk11
+      scala: 2.12.9
```

Comment:
2.12.10 is out now

Reply:
That should probably come in a different PR — tried to upgrade last week (at work) and macro-paradise isn't published for it yet.